src/20240306_AaveV3Ethereum_MeritApprovals/MeritApprovals.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Merit Approvals"
+author: "karpatkey_TokenLogic_ACI"
+discussions: "https://governance.aave.com/t/arfc-merit-a-new-aave-alignment-user-reward-system/16646"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0xc80da83fadfe4f8a4c56e1643895cb7e9b1af1d9dcd374f1b41ded5c95b42f68"
+---
+
+## Simple Summary
+
+“Merit” proposes a merkle-tree-based periodic airdrop system, designed to reward Aave-aligned user behaviors and enhance protocol competitiveness, with a defined budget for the initial 90-day period.
+
+## Motivation
+
+Addressing the competitive DeFi landscape, “Merit” seeks to elevate Aave’s appeal and efficiency through a rewarding mechanism that prioritizes DAO-beneficial actions, distinguishing Aave from competitors employing artificial incentive methods.
+
+## Specification
+
+Merit SAFE: ‘0xdeadD8aB03075b7FBA81864202a2f59EE25B312b’
```

Comment:
same as on any other aip, **please** use etherscan links for addresses so it's easier to check.

Reply:
added